recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
+
+requirements:
+    build:
+        - gcc [osx]
+        - hdf5
+        - cmake
+    run:
+        - libgcc [osx]
+        - hdf5
+
```

Comment:
Does it has any executable commands to test? If not is that a library? In the former you can just use the command with an option for version of help. The latter you can check for the share/static lib file existence. Something like this:

```yaml
test:
    commands:
        - exit $(test -f ${PREFIX}/lib/libGctp.a)  # [not win]
```